tests/wicked/advanced_sut.pm

```diff
@@ -10,6 +10,7 @@
 # Summary: Advanced test cases for wicked
 # Test scenarios:
 # Test 1 : Create a GRE interface from legacy ifcfg files
+# Test 2 : Create a gre interface from wicked XML files
 # Maintainers:
```

Comment:
Travis complains of this line. Missing '# Maintainer: <email address>' in tests/wicked/advanced_sut.pm